STANDARD.md

```diff
@@ -366,41 +366,28 @@ meshes):
                         the *imaginary* part)
                        ![definition of imaginary part](img/cylindrical.png)
 
-  - `dataOrder`
-    - type: *(string)*
-    - description: used for the reading of 1-dimensional arrays of N elements,
-                   where N is the number of dimensions in the simulation;
-                   these should be in the ordering of variation for the indexes
-                   for matrices as defined by the index-operator (`[...][...]`)
-                   of the writing code; can be omitted for 1D records
-    - rationale: in Fortran, ordering of matrices is linearized in memory in
-                 column-major order whereas in C it is row-major; due to that
-                 the index-operators in Fortran and C operate in exactly
-                 opposite order;
-                 supported file-formats keep the order of the writing code's
-                 record components; we still need to store additional
-                 information such as `axisLabels` in a defined order
-    - allowed values:
-      - `C` if data is written by C or a C-like language such as C++, Python,
-        Java
-      - `F` if data is written by a Fortran code
-
   - `axisLabels`
     - type: 1-dimensional array containing N *(string)*
             elements, where N is the number of dimensions in the simulation
-    - description: ordering of the labels for the `geometry` of the mesh
-    - advice to implementors: in the ordering of variation for the indexes for
-                              matrices as defined by the index-operator
-                              (`[...][...]`) of the writing code
-    - advice to implementors: on read, query the record's `dataOrder` to get the
-                              information if you need to invert the access to
-                              `axisLabels` (and other attributes that use the
-                              same definition)
+    - description: this attribute assigns for a matrix `A_{i,j,k}`
```

Comment:
Could you clarify the meaning of `assigns for`, or maybe use some other phrasing?

Reply:
One possibility is 
```
- description: human-reading labels for the `N` axes of the mesh, ordered *from fastest to slowest varying index when accessing the mesh contiguously (as 1D flattened logical memory)`